### PR TITLE
Add new feature count of pending tasks

### PR DESCRIPTION
**Pull request by jaimegarcamo**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -41,9 +46,26 @@ function saveTask(title) {
   localStorage.setItem("tasks", tasks);
 }
 
+function pendingTasks(){
```
Comment (vimtor) [inline src/main.js]: You could name this `updatePendingTasks` to be more clear

File: src/main.js [hunk 2]
```diff
@@ -41,9 +46,26 @@ function saveTask(title) {
   localStorage.setItem("tasks", tasks);
 }
 
+function pendingTasks(){
+  const storedTasks = localStorage.getItem("tasks");
+  let storedTasksArray = [];
+  console.log(storedTasks);
```
Comment (vimtor) [inline src/main.js]: Remove `console.log` from the code. It should only be used for debugging

**Review by vimtor — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 1 above)
Comment (jaimegarcamo) [reply src/main.js]: done

File: src/main.js (hunk 2 above)
Comment (jaimegarcamo) [reply src/main.js]: done

**Review by vimtor — APPROVED** [review]

**Pull request merged**